apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap

```diff
@@ -4892,8 +4902,8 @@ expression: "&schema"
           "description": "The trace ID of the request.",
           "properties": {
             "trace_id": {
-              "$ref": "#/definitions/TraceIdFormat2",
```

Comment:
Looks like there is non-determinism here. I did a check and there are two TraceIdFormat enums. We need to unify them.